src/tests/ftest/util/server_utils.py

```diff
@@ -290,3 +273,35 @@ def kill_server(hosts):
     for host in hosts:
         subprocess.call("ssh {0} \"{1}\"".format(host, '; '.join(kill_cmds)),
                         shell=True)
+
+def clustershell_execute(cmd, hosts, timeout=60):
+    """
+    Python cluster shell execute function with error check.
+    Args:
+        cmd: Command in string to run on cluster
+        hosts: Cluster nodes
```

Comment:
Looking at the usage of this, I'm wondering if it doesn't make more sense to pass in `hosts` as a list and have this function convert that list into a comma-separated string of hosts or [ClusterShell.NodeSet](https://clustershell.readthedocs.io/en/latest/api/NodeSet.html), as required by the `run()` function.

Reply:
done, passing as list from higher function.